.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: JBoss Modules CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  build-test-matrix:
+    name: JDK${{ matrix.jdk }}-${{ matrix.impl}}-${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        jdk: [11]
+        impl: [hotspot, openj9]
+    steps:
+    - uses: actions/checkout@v2
+    # test runs use -Djava8.home=... as well, so we need both JDKs
+    - name: Set up JDK 8
+      uses: AdoptOpenJDK/install-jdk@v1
+      with:
+        version: 8
+        impl: ${{ matrix.impl }}
+    - name: Store JDK8 env (Linux)
+      if: contains(matrix.os, 'ubuntu')
+      run: |
+        echo "::set-env name=JAVA8_HOME::$(echo $JAVA_HOME)"
```

Comment:
Why bother with setting an env var here?  The AdoptOpenJDK install job should set `JAVA_HOME_8` for you already.

Reply:
Well, yeah it does, then installing openjdk 11 overwrites it ;)